roles/webots/directorrobocup.role

```diff
@@ -7,15 +7,21 @@ nuclear_role(
   support::configuration::SoccerConfig
   actuation::KinematicsConfiguration
   support::configuration::GlobalConfig
+  # Network
+  network::NUClearNet
```

Comment:
I think you need overview for nusight to work otherwise I don't think this will do anything